ios/SundarGutka/LaunchScreen.storyboard

```diff
@@ -1,100 +1,56 @@
-                            <constraint firstItem="9FG-es-xrd" firstAttribute="centerX" secondItem="Bcu-3y-fUS" secondAttribute="centerX" id="QUG-nC-Dg2"/>
-                            <constraint firstItem="t4S-X8-0Ap" firstAttribute="centerX" secondItem="Bcu-3y-fUS" secondAttribute="centerX" id="VZw-bv-rpt"/>
-                            <constraint firstItem="GJd-Yh-RWb" firstAttribute="top" secondItem="Bcu-3y-fUS" secondAttribute="top" constant="20" id="cFp-m9-yWK"/>
-                            <constraint firstItem="Bcu-3y-fUS" firstAttribute="bottom" secondItem="uif-bk-E4u" secondAttribute="bottom" id="cgb-4S-mun"/>
-                            <constraint firstItem="Bcu-3y-fUS" firstAttribute="trailing" relation="greaterThanOrEqual" secondItem="MN2-I3-ftu" secondAttribute="trailing" constant="50" id="chL-4Z-A32"/>
-                            <constraint firstItem="MN2-I3-ftu" firstAttribute="centerX" secondItem="9FG-es-xrd" secondAttribute="centerX" id="eXb-BB-eow"/>
-                            <constraint firstItem="Bcu-3y-fUS" firstAttribute="trailing" secondItem="nhk-LM-aeN" secondAttribute="trailing" id="fK4-lq-FQn"/>
-                            <constraint firstItem="nhk-LM-aeN" firstAttribute="top" secondItem="MN2-I3-ftu" secondAttribute="bottom" constant="8" symbolic="YES" id="h83-tu-peb"/>
-                            <constraint firstItem="MN2-I3-ftu" firstAttribute="leading" relation="greaterThanOrEqual" secondItem="Bcu-3y-fUS" secondAttribute="leading" constant="50" id="o48-AI-utB"/>
-                            <constraint firstItem="MN2-I3-ftu" firstAttribute="top" secondItem="9FG-es-xrd" secondAttribute="bottom" constant="8" symbolic="YES" id="qGk-IG-csk"/>
-                            <constraint firstItem="GJd-Yh-RWb" firstAttribute="centerX" secondItem="Bcu-3y-fUS" secondAttribute="centerX" id="sTS-6o-s37"/>
-                            <constraint firstItem="nhk-LM-aeN" firstAttribute="centerX" secondItem="Bcu-3y-fUS" secondAttribute="centerX" id="x2X-nj-P7M"/>
-                            <constraint firstItem="uif-bk-E4u" firstAttribute="centerX" secondItem="Bcu-3y-fUS" secondAttribute="centerX" id="zuG-OH-h5T"/>
+                            <constraint firstItem="083-jd-HN7" firstAttribute="centerY" secondItem="Ze5-6b-2t3"
+                                secondAttribute="centerY" id="Kzf-hd-nYn" />
+                            <constraint firstItem="083-jd-HN7" firstAttribute="centerX" secondItem="Ze5-6b-2t3"
+                                secondAttribute="centerX" id="OhI-II-7PK" />
                         </constraints>
                     </view>
                 </viewController>
-                <placeholder placeholderIdentifier="IBFirstResponder" id="iYj-Kq-Ea1" userLabel="First Responder" sceneMemberID="firstResponder"/>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="iYj-Kq-Ea1" userLabel="First Responder"
```

Comment:
Veerji, I recommend you to use Airbnb or any good eslint specification. Though, that can be added in next PR but here's a quick tip.

Maybe you should break down the props of placeholder to new line if it doesn't allow you go keep all of them in same like. Example:

```js
<placeholder 
  placeholderIdentifier="IBFirstResponder"
  id="iYj-Kq-Ea1"
  userLabel="First Responder"
  sceneMemberID="firstResponder" 
/>
```

It will make the code look more readable.

Reply:
veer Ji this is the IOS folder code which is auto generated by react-native so fixing indentation doesn't automatically work here. I can do it manually but that's not the right way.

Reply:
No worries veerji, ignore this one then.